zookeeper-contrib/zookeeper-contrib-zooinspector/pom.xml

```
@@ -91,13 +91,9 @@
     </dependency>
     <dependency>
       <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-core</artifactId>
-      <exclusions>
-        <exclusion>
-          <groupId>*</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-      </exclusions>
+      <artifactId>logback-classic</artifactId>
+      <scope>runtime</scope>
+      <optional>true</optional>
```

To avoid propagation, it was decided on #2155 that optional runtime dependencies should be added as a test dependency when they are needed for tests, to prevent them being propagated when used as a library, rather than using optional:

```suggestion
      <scope>test</scope>
```